### PR TITLE
Reorganization: Unfettered with their own files

### PR DESCRIPTION
**Pull request by Hurleveur**
This only moves content around, no changes were made. (same amount of lines except for the 3 x 13 lines of copyrights + 2 newlines)

With HR getting merged the hai files got bigger, and this will help clarify things for the Unfettered campaign PR (which does move  these missions around too). Currently it may not seem like there are many Unfettered jobs and missions but I will add many soon... I did not want to PR this before HR got merged to not make conflicts, but everything should be good now.

I'll be opening a PR to add new jobs for the Unfettered next, this will help make things clear. I'll also open a fix to the "it" used to refer to them, instead of "they".

### CODE REVIEW

**Comment by Hurleveur** [comment]
Side note: I did not move the Hai fleets around since its such a small file and I'm not intending on making it bigger.
I was careful to just move things around by copy pasting.

I could separate other things, or even give them their own subfolder, do feel free to suggest changes.

**Review by warp-core — COMMENTED** [review]
I know all the new Jump Shield Beetle variants were used by the Unfettered, but they appeared in *Hai* Reveal (arguably, they should be defined in the Hai Reveal files, are they ever used outside of missions?)

**Comment by Hurleveur** [comment]
They're not used outside of missions so I guess I'll move them in their respective missions?
Could also have a hai variants.txt file

**Comment by EjoThims** [comment]
With Unfettered coming too, I'd think it's worth getting the Hai a variant file while we're moving things around anyway.

**Comment by Hurleveur** [comment]
Thats a worthwile change but it can and should be in another PR, this one just aims to make UHai campaign easier to read, by not making it change so much for these jobs and missions.

~~I'll probably open that other variant PR myself if I cant sleep~~

**Comment by Amazinite** [comment]
Splitting mission files makes sense to me (human missions of different factions/regions are in different files), but splitting ship files doesn't (all human or Coalition or Korath or Wanderer ships are in the same file; it's not like we split by government or subfaction within a species elsewhere).

File: data/hai/hai jobs.txt [hunk 1]
```diff
@@ -298,205 +298,6 @@ mission "Hai Festival [3]"
...
-	repeat
-	clearance "After speaking with the Unfettered and explaining your presence, they give you permission to land with the cargo."
-	description "Deliver <tons> of food to the Unfettered on <destination>. You will be paid <payment>."
-	cargo "food (aid)" 25 2 .05
-	to offer
-		random < 40
-		"cargo space" >= 50
-		has "Ask the Hai about the Unfettered: offered"
-	source
-		government "Hai"
```
Comment (MasterOfGrey) [inline data/hai/hai jobs.txt:327]: This applies to all missions of this type.

Jobs/missions which offer in Hai space, should be in Hai files, even if they're related to the Unfettered.

Unfettered files should be for jobs or missions that offer in Unfettered space, or if such missions were to exist - potentially from Unfettered agents in Hai space. 

These aid missions are *Hai* missions.

**Review by MasterOfGrey — CHANGES_REQUESTED** [review]

**Comment by Hurleveur** [comment]
> Splitting mission files makes sense to me (human missions of different factions/regions are in different files), but splitting ship files doesn't (all human or Coalition or Korath or Wanderer ships are in the same file; it's not like we split by government or subfaction within a species elsewhere).

I did it for clarity, not any other reason. The Korath ships.txt is a mess because everything is in it, and I'll add 5 ships in part 1 of the campaign alone, + at least 2 in part 2, so it'll get real messy.
I'll cancel those changes if you disagree with them, though.

**Pull request merged**